packages/web-app-files/src/composables/actions/spaces/useSpaceActionsUploadImage.ts

```diff
@@ -39,7 +38,7 @@ export const useSpaceActionsUploadImage = ({
 
   const uploadImageSpace = async (ev) => {
     const graphClient = clientService.graphAuthenticated
-    const file = ev.currentTarget.files[0]
+    const file = ev.currentTarget.files[0] as File
```

Comment:
Cant you type ev properly instead?

Reply:
i decided to remove it again since we could also have, lastModifiedDate, name, ...